src/components/Output.js

```diff
@@ -1,20 +1,22 @@
-/* eslint-disable react/prop-types */
-/* eslint-disable react/prefer-stateless-function */
 import React from 'react';
+import PropTypes from 'prop-types';
 
-export default class Output extends React.Component {
-  render() {
-    const { total, next, operation } = this.props;
-    return (
-      <div className="output">
-        <div className="operand">
-          <p>
-            { total }
-            { operation }
-            { next }
-          </p>
-        </div>
+export default function Output(props) {
+  Output.propTypes = {
```

Comment:
- [OPTIONAL] Following ES6 syntax, you could implement it to your other components too. ♻️

Reply:
Actioned :heavy_check_mark: